### PR TITLE
Update faqs.md

### PR DESCRIPTION
**Pull request by epowell101**
Some slightly nicer wording inviting participation and contribution which is fundamental to our success..  Also the first answer literally answered the technology used to keep the data safe however there was not a summary statement - so this was added.

There may be typos and some link formatting is probably incorrect.

### CODE REVIEW

File: quickstart/faqs.md [hunk 1]
```diff
@@ -1,62 +1,70 @@
...
...ity to enhance the durability of data at rest within whatever physical persistence layer is in use.  When volumes are provisioned which are configured for replication \(a user can control the count of active replicas which should be maintained, on a per StorageClass basis\), write I/O operations issued by an application to that volume are 'amplified' by the nexus and sent to all active replicas.  Only if every replica completes the write successfully on its own underlying block device will the I/O completion be acknowledged.  Otherwise, the I/O is failed and must be retried by the caller.  If a replica is determined to have faulted \(I/O cannot be serviced within the configured timeout period, or not without error\), the control plane will automatically take corrective action and remove it from the volume.  If spare capacity is available within another Mayastor pool, a new replica will be created as a replacement and automatically brought into syncrhonisation with the existing replicas.  In short, Mayastor adds availability and consistency for data through ensuring that a minimum set of replicas ara available and by recovering from outages to retain that minimum number of replicas.
```
Comment (AmitKumarDas) [inline quickstart/faqs.md:6]: spellcheck `syncrhonisation`

Comment (AmitKumarDas) [reply quickstart/faqs.md:6]: spellcheck `minimum set of replicas **ara** available`

Comment (AmitKumarDas) [reply quickstart/faqs.md:6]: It might be nice if we have shorter sentences.

File: quickstart/faqs.md [hunk 2]
```diff
@@ -1,62 +1,70 @@
...
+Mayastor has been built to deliver as close as possible to local disk performance with a specific design goal of less than 10% overhead versus the performance capabilities of underlying devices including always improving high performance SSDs; while this may sound like common sense, <10% overhead is much lower overhead than traditional storage systems and storage software which often impart 40-50% or greater overhead. For this reason, the Mayastor I/O path is predicated on NVMe, a transport which is both highly CPU efficient and which demonstrates highly linear resource scaling. The data path runs entirely within user space, also contributing efficiency gains as syscalls are avoided, and is both interrupt and lock free; in other words, Mayastor like many high performance networking systems bypasses the kernel in order to deliver higher performance.
```
Comment (AmitKumarDas) [inline quickstart/faqs.md:19]: Can we also provide the core data structure(s) that this storage system relies on? For example, is it BTree or LSM or something else. Are there any reference papers that has inspired the development of this storage system?

File: quickstart/faqs.md (hunk 1 above)
Comment (GlennBullingham) [reply quickstart/faqs.md:6]: Why have we removed the link to the data path?  Does this add not add value?

File: quickstart/faqs.md (hunk 2 above)
Comment (GlennBullingham) [reply quickstart/faqs.md:19]: I have issues with the factual correctness of some assertions:
 - including always improving high performance SSDs;
 - Mayastor like many high performance networking systems bypasses the kernel

In the first case, that's just not true, as an absolute statement.  We know already that there is some overhead, and for write intensive workloads, it is significant.  It certainly doesn't "always" improve.

Mayastor might bypass the kernel but the I/O path in general does not, not in its entirety.  It's still dependent on the kernel initiator of the worker node which is mounting the volume.

Comment (GlennBullingham) [reply quickstart/faqs.md:19]: On structures, this question is already asked (and answered) specifically in another question - "What is the on-disk format"

**Pull request closed without merging**